content/en/blog/_posts/2020-12-08-kubernetes-release-1.20.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: blog
+title: 'Kubernetes 1.20: The Raddest Release'
+date: 2020-12-08
+slug: kubernetes-1-20-release-announcement
+---
+
+**Authors:** [Kubernetes 1.20 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.20/release_team.md)
+
+We’re pleased to announce the release of Kubernetes 1.20, our third and final release of 2020! This release consists of 44 enhancements: 11 enhancements have graduated to stable, 15 enhancements are moving to beta, and 16 enhancements are entering alpha.
+
+The 1.20 release cycle returned to its normal cadence of 11 weeks following the previous extended release cycle. This is one of the most feature dense releases in a while: the Kubernetes' innovation cycle is still trending upward. This release has more Alpha vs Stable enhancements, showing that there is still a lot to be explored in the cloud native ecosystem.
+
+## Major Themes
+
+### Volume Snapshot Operations Goes Stable
+
+This feature provides a standard way to trigger volume snapshot operations and allows users to incorporate snapshot operations in a portable manner on any Kubernetes environment and supported storage providers.
+
+Additionally, these Kubernetes snapshot primitives act as basic building blocks that unlock the ability to develop advanced, enterprise-grade, storage administration features for Kubernetes, including application or cluster level backup solutions.
+
+Note that snapshot support will require Kubernetes distributors to bundle the Snapshot controller, Snapshot CRDs, and validation webhook. In addition, a CSI driver supporting the snapshot functionality must also be deployed on the cluster.
+
+### Kubectl Debug Graduates to Beta
+
+`Kubectl alpha debug` graduates to beta in 1.20, becoming `kubectl debug`. `Kubectl debug` provides support for common debugging workflows directly from kubectl. Troubleshooting scenarios supported in this release of kubectl include:
```

Comment:
Do we capitalize commands at the beginning of sentences? - Didn't see a definite answer in Docs [style guide](https://kubernetes.io/docs/contribute/style/style-guide/)

Reply:
I went back and forth on this and couldn't find a reference.

Reply:
@zacharysarah @jimangel do either of you have thoughts?

Reply:
@sethmccombs not sure if my opinion matter that much :-) but I recomend doing lowercase.  It looks better, I think.

Reply:
We definitely shouldn't artificially capitalize a string literal.
```suggestion
The `kubectl alpha debug` features graduates to beta in 1.20, becoming `kubectl debug`. The feature provides support for common debugging workflows directly from kubectl. Troubleshooting scenarios supported in this release of kubectl include:
```

Reply:
lower cased the commands.